modules/aws/document_db/main.tf

```diff
@@ -0,0 +1,70 @@
+module "security_group" {
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "~> 4"
+
+  name        = var.name
+  description = "Complete DocumentDB security group"
+  vpc_id      = var.vpc.vpc_id
+
+  # ingress
+
+  ingress_with_cidr_blocks = var.visibility == "public" ? [
```

Comment:
1. should be accessible from within vpc for private and not externally
2. should be accessible from within vpc and externally for public
3. switching between private and public should work